lib/ui/pages/coursetable/course_table_page.dart

```diff
@@ -651,7 +651,7 @@ class _CourseTablePageState extends State<CourseTablePage> {
             Text(sprintf("%s : %s", [R.current.instructor, teacherName])),
           ],
         ),
-        actions: courseInfo.main.course.id.isNotEmpty
+        actions: (courseInfo.main.course.id.isNotEmpty)
```

Comment:
```suggestion
        actions: courseInfo.main.course.id.isNotEmpty
```
這個括號應該是多餘的，`dart format` 也沒有意見。

Reply:
ok